docs/apm/getting-started.asciidoc

```diff
@@ -1,22 +1,45 @@
 [role="xpack"]
 [[apm-getting-started]]
-== Getting Started
+== Get started with APM
 
-If you have not already installed and configured Elastic APM,
-the *Setup Instructions* will get you started.
+++++
+<titleabbrev>Get started</titleabbrev>
+++++
 
-[role="screenshot"]
-image::apm/images/apm-setup.png[Installation instructions on the APM page in Kibana]
+Elastic APM captures different types of information from within instrumented applications:
 
+* *Spans* contain information about a specific code path that has been executed.
+They measure from the start to end of an activity,
+and they can have a parent/child relationship with other spans.
+* *Transactions* are a special kind of span that have extra metadata associated with them.
```

Comment:
Can we clearly call out that Transaction is essentially the first span for a particular service rather than call it the highest level of work?